plugins/Sitemaps/class.sitemaps.plugin.php

```diff
@@ -110,7 +110,7 @@ public function utilityController_robots_create($sender) {
         $sender->deliveryType(DELIVERY_TYPE_VIEW);
         $sender->setHeader('Content-Type', 'text/plain');
 
-        $robots = new \Vanilla\Sitemaps\Robots();
+        $robots = new \Vanilla\Sitemap\Robots();
```

Comment:
The namespace really should match the name of the addon which is plural.

Reply:
In any event, we've moved this class up into core now. I think the class is now `Vanilla\Web\Robots`.